code/obj/item/organ_holder.dm

```diff
@@ -1616,3 +1616,17 @@
 			src.icon_state = initial(src.icon_state)
 		else
 			src.icon_state = "[initial(src.icon_state)]_cd"
+
+/datum/targetable/organAbility/sacredheart
+	name = "Sacred Heart"
+	desc = "Channel your faith in god into a knife to smite your enemies."
+	icon_state = "cyberkidney"
+	targeted = 0
+	cooldown = 30 SECONDS
+
+	cast(atom/target)
+		if (..())
+			return 1
+		boutput(holder.owner, "<span class='notice'>You use your [islist(linked_organ) ? "s" : ""] to conqure a knife from thin air.</span>")
```

Comment:
in what case will this be a list

Reply:
probably none, It was just a holdover from other organ code